test/version/nmstate.go

```diff
@@ -0,0 +1,35 @@
+package version
+
+import (
+	"fmt"
+	"regexp"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Masterminds/semver"
+
+	"github.com/nmstate/kubernetes-nmstate/test/runner"
+)
+
+var (
+	r = regexp.MustCompile(`nmstate-(.*)-.*`)
+)
+
+func IsNmstate(constraint string) bool {
+	rpmVersion := runner.RunAtFirstHandlerPod("rpm", "-qe", "nmstate")
```

Comment:
Just a thought, shouldn't we worry about different nmstate versions on handlers?

Reply:
Right now nmstate 1.x is retro-compatible enough so it understand nmstate-0.3 schema too.

Reply:
Thanks for explaining